src/main/java/core/basesyntax/service/impl/DataConverterImpl.java

```
@@ -0,0 +1,52 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.service.DataConverter;
+import java.util.ArrayList;
+import java.util.List;
+
+public class DataConverterImpl implements DataConverter {
+
+    @Override
+    public List<FruitTransaction> convertStringsToTransactionsList(List<String> lines) {
+        if (lines == null) {
+            throw new NullPointerException("Input parameter cannot be null");
+        }
+        List<FruitTransaction> fruitTransactions = new ArrayList<>();
+        for (int i = 1; i < lines.size(); i++) {
```

The method skips the first line by starting the loop at index 1 (for (int i = 1; i < lines.size(); i++)), which assumes a header is always present. The task requires: "Skip or handle the CSV header line on read/parse." Ensure you validate that a header exists (or handle the case when the file has no header) and fail fast with an informative message if the header is missing or malformed.